frontend/packages/process-editor/src/components/Canvas/Canvas.module.css

```diff
@@ -2,9 +2,14 @@
   --mask-icon-size: 0 0 /46px 46px;
 }
 
+.container {
+  flex: 3;
+}
+
 .wrapper {
   display: flex;
   flex: 1;
+  height: calc(90vh - var(--header-height));
```

Comment:
Not sure, but should we move the height to `.container` and then change the height on the wrapper to `height: 100%`? 🤔 I believe it should then spend 100% of its parent element which is the container element. 🤔 But I'm not sure about this suggestion. 🙈

Reply:
Yep, that worked!